src/System.Management.Automation/resources/HostInterfaceExceptionsStrings.resx

```diff
@@ -121,7 +121,7 @@
     <value>An error of type "{0}" has occurred.</value>
   </data>
   <data name="HostFunctionNotImplemented" xml:space="preserve">
-    <value>A command that prompts the user failed because the host program or the command type does not support user interaction. Try a host program that supports user interaction, such as the Windows PowerShell Console or Windows PowerShell ISE, and remove prompt-related commands from command types that do not support user interaction, such as Windows PowerShell workflows.</value>
+    <value>A command that prompts the user failed because the host program or the command type does not support user interaction. Try a host program that supports user interaction, such as the PowerShell Console, and remove prompt-related commands from command types that do not support user interaction, such as PowerShell workflows.</value>
```

Comment:
Remove mention of workflows which isn't supported

Reply:
Fixed.